website/docs/r/azure_secret_backend.html.md

```diff
@@ -36,6 +36,16 @@ resource "vault_azure_secret_backend" "azure" {
 }
 ```
 
+```hcl
```

Comment:
```suggestion
You can setup the Azure secrets engine with Workload Identity Federation (WIF) for a secret-less configuration:
```hcl
```

Reply:
Also, maybe we should move this up as the first example so that it is more prominent? I think we will want to make WIF our recommended approach for configuration.